python/api/main.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python
+
+from typing import List
+import functions_framework
+from flask import Request
+from cdp_backend.database import models as db_models
+import fireo
+from google.auth.credentials import AnonymousCredentials
+from google.cloud.firestore import Client
+from flask import jsonify
+
+
+@functions_framework.http
+def get_event_source_ids(request: Request):
+    fireo.connection(
+        client=Client(
+            project="cdp-montana-legislature",
+            credentials=AnonymousCredentials(),
```

Comment:
Is it OK to use AnonymousCredentials? This would be easiest by far...

Reply:
Yes